exe/floe

```diff
@@ -31,7 +31,7 @@ runner_options = opts[:docker_runner_options].to_h { |opt| opt.split("=", 2) }
 
 Floe::Workflow::Runner.docker_runner = runner_klass.new(runner_options)
 
-context = Floe::Workflow::Context.new(:input => opts[:input])
+context = Floe::Workflow::Context.new(:input => JSON.parse(opts[:input]))
```

Comment:
@kbrock what do you think about moving this into the `Context` initializer?  We already do a similar `context = JSON.parse(context) if context.kind_of?(String)` for the context if it is passed in.  Otherwise all callers would have to do this e.g. https://github.com/ManageIQ/manageiq-providers-workflows/blob/master/app/models/manageiq/providers/workflows/automation_manager/workflow.rb#L14

Reply:
Is a string valid json input?
Can we assume we only want a hash (though I guess array could work)

Reply:
> When a state machine is started, the caller can provide an initial [JSON text](https://tools.ietf.org/html/rfc8259#section-2) as input, which is passed to the machine's start state as input.

I believe we can assume that input must be JSON per the spec